src/insns/cmv_16bit.adoc

```diff
@@ -1,21 +1,19 @@
 <<<
-//[#insns-cmove-16bit,reftext="Capability move (C.MV, CMove), 16-bit encoding"]
 
 [#C_MV,reftext="C.MV"]
 ==== C.MV
-See <<C.CMOVE>>.
-
-[#C_CMOVE,reftext="C.CMOVE"]
-==== C.CMOVE
 
 Synopsis::
-Capability move (C.MV, C.CMOVE), 16-bit encoding
+Capability move (C.MV), 16-bit encoding
 
 Capability Mode Mnemonic::
-c.cmove cd, cs2
+c.mv cd, cs2
 
 Capability Mode Expansion::
-cmove cd, cs2
+cmv cd, cs2
+
+NOTE: the mnemonic must be unique, but the intention is to always use `mv` in the
```

Comment:
```suggestion
NOTE: the instruction names must be unique, but the intention is to always use `mv` in the
```

I wonder if we should be using instruction name instead of mnemonic here? To me mnemonic makes me think of to the assembler syntax.